.github/workflows/shared-release.yml

```diff
@@ -83,7 +83,7 @@ jobs:
           fi
 
           if [[ $PREVIOUS_VERSION ]]; then
-            SINCE_ARG="--since-tag $PREVIOUS_VERSION"
+            OUTPUT_ONLY_CURRENT_VERSION=true
           fi
```

Comment:
Nit: maybe include `IS_DEV_RELEASE` in the check? I know it's not strictly necessary, but it helps in understanding the code path here since this issue only affects dev release changelogs if I understand correctly.

E.g.

```bash
if [ $IS_DEV_RELEASE ] && [ $PREVIOUS_VERSION ]; then
```

Reply:
I don't think it is correct to depend on dev releases, since we want GitHub releases for `dev` and proper version of Rust. We could make it dependent  on `inputs.create-github-release`, because it should only be needed then. Also the changelog generator step should depend on that then. @cycraig what do you think?

Reply:
I thought release notes including changelogs from multiple versions was only an issue for dev-releases though?

I understand it's more of a formatting issue for all releases now though, since it also removes the version header. I think `inputs.create-github-release` makes sense. 👍